query/sources/common/src/main/java/org/codice/ddf/admin/sources/fields/CswSpatialOperator.java

```diff
@@ -51,209 +50,281 @@ public CswSpatialOperator(DataType<String> spatialOperator) {
                 spatialOperator);
     }
 
-    protected static final class NoFilter extends StringField {
+    public static final class NoFilter implements EnumValue<String> {
 
-        public static final String OPERATOR = "NO_FILTER";
+        public static final String ENUM_TITLE = "None";
 
-        public static final String NO_FILTER_FIELD_NAME = "None";
+        public static final String OPERATOR = "NO_FILTER";
 
         public static final String DESCRIPTION =
                 "Indicates that no spatial operators should be applied.";
 
-        public NoFilter() {
-            super(NO_FILTER_FIELD_NAME, NO_FILTER_FIELD_NAME, DESCRIPTION);
+        @Override
+        public String enumTitle() {
+            return ENUM_TITLE;
```

Comment:
I believe we'll experience enums with values that differ from their titles much more often in the future. We experienced this when playing around with the catalog. The metacard attributes had periods which are prohibited as enum titles. As for whether there will ever be enums other than strings, it will likely be a rare case but I didn't see any harm in supporting. It was an extremely minimal amount of code to support it.. I don't mind making the abstract class that you mentioned if we think it's valuable enough.